src/test/groovy/org/mockito/release/notes/vcs/DefaultCommitApproverTest.groovy

```diff
@@ -0,0 +1,52 @@
+package org.mockito.release.notes.vcs
+
+import spock.lang.Specification
+
+class DefaultCommitApproverTest extends Specification {
```

Comment:
Its all about what we call unit, but in this case I agree that we can later have more approvers and its better to cover every class in different tests. I will fix it.